src/libraries/System.Security.Cryptography.X509Certificates/tests/Cert.cs

```diff
@@ -86,5 +86,10 @@ public void Dispose()
         }
 
         private X509Certificate2[] _certs;
+
+        public static implicit operator ImportedCollection(X509Certificate2Collection collection)
```

Comment:
I'm surprised to see this coming from you :)

Is it really necessary to add this implicit cast?

Reply:
> Is it really necessary to add this implicit cast?

Nah.  Added the extra words and removed it.

(Probably what I wanted was to put an accelerator for Find on the wrapper class, but instead I just added the ctor-wraps.)